### DISCUSSION TITLE
Documentation missing?

### DISCUSSION THREAD

**Discussion by JdMDE** [MTG/essentia#1424] [Q&A] [answered]
Sorry for the annoyance of a newbie but I was recently programming in C++ with libessentia and, after retaking my work, I can't find the documentation. The intended page, https://essentia.upf.edu/ seems to be empty.
 I found the HTML code-generated documentation recently, but not now. Am I looking for in the wrong place?
  Thanks a lot for your time.

**Comment by palonso** [comment, marked as answer]
@JdMDE ,
You are looking at the right place!

Our website is hosted at our university's infrastructure and we noticed problems this evening.
We have experienced similar issues before and normally they don't last longer than a few hours.

**Reply by JdMDE** [reply]
Everything works fine now. Thanks a lot for your response.
